ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh

```diff
@@ -132,6 +132,9 @@ azurestack)
     export TEST_PROVIDER="none"
     export AZURE_AUTH_LOCATION=${SHARED_DIR}/osServicePrincipal.json
     export SSL_CERT_FILE="${CLUSTER_PROFILE_DIR}/ca.pem"
+    export AZURE_CLIENT_ID="$(cat ${AZURE_AUTH_LOCATION} | jq -r .clientId)"
+    export AZURE_CLIENT_SECRET="$(cat ${AZURE_AUTH_LOCATION} | jq -r .clientSecret)"
```

Comment:
Just a thought, can you use AZURE_AUTH_LOCATION by itself in origin, and then read the file? If it's avoidable, it's better not to put any secrets in environment variables directly since you increase the chance of a leak somewhere along the way

Reply:
We are using azure golang sdk and those are the expected envs. We can read the files inside origin and feed it to the library if that is preferred. But this pattern has been used in quite a few places in the repo: https://github.com/search?q=repo%3Aopenshift%2Frelease%20AZURE_AUTH_LOCATION&type=code

Reply:
Those are invoking `az` directly so it's slightly different, but it's fine either way if you prefer this

Reply:
I am going to experiment in origin as well. If everything works well, I will come back and remove the extra envs.